MeshLib/IO/MPI_IO/NodePartitionedMeshReader.cpp

```diff
@@ -217,9 +219,11 @@ void NodePartitionedMeshReader::readPropertiesBinary(
     std::ifstream is(fname_cfg.c_str(), std::ios::binary | std::ios::in);
     if (!is)
     {
-        WARN("Could not open file '%s'.\n"
-             "\tYou can ignore this warning if the mesh does not contain %s-"
-             "wise property data.", fname_cfg.c_str(), item_type.data());
+        WARN(
+            "Could not open file '{:s}'.\n"
+            "\tYou can ignore this warning if the mesh does not contain {:s}-"
+            "wise property data.",
+            fname_cfg.c_str(), item_type.data());
```

Comment:
`item_type` is already a `string`. Maybe, calling the method `data()` isn't necessary?

Reply:
Same as c_str